ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-nightlies-postsubmits.yaml

```diff
@@ -417,6 +417,52 @@ postsubmits:
     - ^nightlies$
```

Comment:
Seems like you also need to make changes in
1. `ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-postsubmits.yaml`
2. `ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml`